spec/pushmi_pullyu/swift_depositer_spec.rb

```diff
@@ -59,9 +59,21 @@
         )
         expect(swift_depositer).not_to be_nil
         # rubocop:disable Layout/LineLength
-        expect(swift_depositer.swift_connection.connection.authtoken).to eq('gAAAAABl8hYAouKZJLkt8NDmuA2NjA1zOasGOAX-b2MfKpjiM_kf8sZHe42ipcs6Vb-57-aATajbTg54wIwhNhl2HKRfz5_rKfSJ0PnBQNFCVd4bKrdC0pHzoJMn9hkAa2tjBkqppBcMayvfqz-Ppxn0USnHw0z9zLLKDxGbRZwyhDJDhGOcIZg')
+        expect(swift_depositer.instance_variable_get('@swift_connection').connection.authtoken).to eq('gAAAAABl8hYAouKZJLkt8NDmuA2NjA1zOasGOAX-b2MfKpjiM_kf8sZHe42ipcs6Vb-57-aATajbTg54wIwhNhl2HKRfz5_rKfSJ0PnBQNFCVd4bKrdC0pHzoJMn9hkAa2tjBkqppBcMayvfqz-Ppxn0USnHw0z9zLLKDxGbRZwyhDJDhGOcIZg')
         # rubocop:enable Layout/LineLength
       end
     end
+
+    it 'uses ruby-openstack gem behaviour to refresh authentication token' do
+      VCR.use_cassette('swift_new_deposit') do
```

Comment:
For repeatability this might need to be a different tape than swift_new_deposit.  Also maybe we should setup the docker container to set the `token_life`?